xonsh/xontribs.json

```diff
@@ -43,6 +43,15 @@
   "url": "https://github.com/AstraLuma/xontrib-avox",
   "description": ["Automatic (de)activation of virtual environments as you cd around"]
  },
+ {"name": "back2dir",
+  "package": "xontrib-back2dir",
+  "url": "https://github.com/anki-code/xontrib-back2dir",
+  "description": [
+    "Back to the latest used directory when starting xonsh shell. ",
```

Comment:
Maybe "Return to the previously used" or "Return to the most recently used..." is better than "Back to..."